fsh-generated/fsh-index.json

```diff
@@ -23,12 +23,28 @@
     "startLine": 1,
     "endLine": 10
   },
+  {
+    "outputFile": "StructureDefinition-au-erequesting-imag-request.json",
+    "fshName": "AUeRequestingImagingServiceRequest",
+    "fshType": "Profile",
+    "fshFile": "au-erequesting-imag-request.fsh",
+    "startLine": 1,
+    "endLine": 15
+  },
+  {
+    "outputFile": "StructureDefinition-au-erequesting-path-request.json",
```

Comment:
the id if following the conventions in base and core would be 'au-erequesting-servicerequest-path'